rust/Cargo.toml

```diff
@@ -8,9 +8,12 @@ members = [
     "common/alloc",
     "common/cookieless",
     "common/dns",
+    "common/flag-analytics",
```

Comment:
this feels a bit specific to flags (it's in the name). why does this make sense as common? the downside of common dependencies is that they compile for everyone (the upside being the shared code and DRY and all that). But I don't really understand the use case of moving this.